jsapp/js/api/mutation-defaults/user-team-organization-usage.ts

```diff
@@ -0,0 +1,307 @@
+import {
+  getOrganizationsInvitesCreateMutationOptions,
+  getOrganizationsInvitesDestroyMutationOptions,
+  getOrganizationsInvitesListQueryKey,
+  getOrganizationsInvitesPartialUpdateMutationOptions,
+  getOrganizationsInvitesRetrieveQueryKey,
+  getOrganizationsListQueryKey,
+  getOrganizationsMembersDestroyMutationOptions,
+  getOrganizationsMembersListQueryKey,
+  getOrganizationsMembersPartialUpdateMutationOptions,
+  getOrganizationsMembersRetrieveQueryKey,
+  getOrganizationsPartialUpdateMutationOptions,
+  getOrganizationsRetrieveQueryKey,
+  type organizationsInvitesListResponse,
+  type organizationsInvitesRetrieveResponse,
+  type organizationsMembersListResponse,
+  type organizationsMembersListResponse200,
+  type organizationsMembersRetrieveResponse,
+} from '#/api/react-query/user-team-organization-usage'
+import { queryClient } from '#/query/queryClient'
+import session from '#/stores/session'
+import { getAssetUIDFromUrl } from '#/utils'
+import { filterListSnapshots, onErrorRestoreSnapshots, onSettledInvalidateKeys } from './common'
+
+queryClient.setMutationDefaults(
+  getOrganizationsPartialUpdateMutationOptions().mutationKey!,
+  getOrganizationsPartialUpdateMutationOptions({
+    mutation: {
+      onSettled: (_data, _error, variables) => {
+        queryClient.invalidateQueries({ queryKey: getOrganizationsListQueryKey() })
```

Comment:
I wonder if it would be nice to add short one line comment before every `queryClient.invalidateQueries(` line to make the file read faster. WDYT?

Reply:
Like here we could have:

```ts
// After an organization is updated, we invalidate all lists of organizations
```

Reply:
Because I find it very useful to read the comments you've added already :)